Types.h

```diff
@@ -3,8 +3,8 @@
 
 typedef unsigned char Byte;
 typedef Byte Bool;
-typedef unsigned short Word;
-typedef unsigned short ObjectPointer;
+typedef unsigned int Word;
```

Comment:
I think, unless there are other problems associated with using `short` here, it's best to leave these typedefs alone and just change the bitfield definitions in e.g. `Interpreter_Constants.h`. Those structs never get used in the code, and the assumption that a Word is 16 bits wide is baked deeply into the Blue Book and this implementation.

Reply:
ideally `Word` would be a `uint16_t` but that is a very recent addition to the language.